tests/test_without_ipywidgets.py

```diff
@@ -11,8 +12,14 @@ def test_interactive():
 
     cost = LeastSquares([1.1, 2.2], [3.3, 4.4], 1, lambda x, a: a * x)
 
-    iminuit.Minuit(cost, 1).interactive()
+    with patch("IPython.get_ipython") as mock_get_ipython:
+        mock_shell = MagicMock()
+        mock_shell.__class__.__name__ = "ZMQInteractiveShell"
+        mock_shell.config = {"IPKernelApp": True}
+        mock_get_ipython.return_value = mock_shell
```

Comment:
If you turn this code into a fixture like I suggested in test_draw, we can reuse it here without repeating the code.